lib/oli/delivery/sections.ex

```diff
@@ -1380,46 +1378,27 @@ defmodule Oli.Delivery.Sections do
         "28" => 43,
       }
   """
-
-  def get_resource_to_container_map(section) do
-    case section do
-      %Section{resource_to_container_map: nil} ->
-        Logger.warning(
-          "Section #{section.slug} has no precomputed resource_to_container_map. One will be generated now."
-        )
-
-        {:ok, section} = update_resource_to_container_map(section)
-        section.resource_to_container_map
-
-      %Section{resource_to_container_map: resource_to_container_map} ->
-        resource_to_container_map
-    end
-  end
-
-  @doc """
-  Builds a section's resource_to_container_map and updates the section with it.
-  """
-  def update_resource_to_container_map(section) do
-    update_section(section, %{
-      resource_to_container_map: build_resource_to_container_map(section)
-    })
```

Comment:
Change caching strategy from storing in the section record to using standardized SectionCache cachex